src/components/Navigation.js

```diff
@@ -29,8 +29,8 @@ export const navigationCSS = ({ interactionIsIdle }: NavState) => ({
 });
 
 export const Navigation = (props: NavProps) => {
-  const { children, getStyles, isFullscreen, isModal } = props;
-  return !isTouch() ? (
+  const { children, getStyles, isFullscreen, isModal, hideNavigationOnTouchDevice } = props;
+  return !isTouch() || !hideNavigationOnTouchDevice ? (
```

Comment:
I've thought about this, and this doesn't really work. Because on Desktop, when `isTouch()` is `false`, the Navigation will always be visible. So even if `hideNavigationOnTouchDevice`.

I could include the flag `hideNavigationOnTouchDevice` into the `isTouch()` function to make it work.. but then I though, is `isTouch()` even still needed here? I think depending on what the user sets, it should taken regardless of `isTouch()`. And as this prop is default `true`, the navigation will be hidden per default, which means the backwards compatibility is given.

Reply:
Yes, we would want the prop to override `isTouch` and we'd also want the backwards compatibility.

I'll have a look at this.